src/wp-includes/block-supports/block-style-variations.php

```diff
@@ -254,22 +250,8 @@ function wp_resolve_and_register_block_style_variations( $variations ) {
 		 * need to have their name set to the kebab case version of their title.
 		 */
 		$variation_name  = $have_named_variations ? $key : _wp_to_kebab_case( $variation['title'] );
```

Comment:
Nit: There's a double space here that could be removed to align the `=` properly.

Reply:
Fixed at https://github.com/WordPress/wordpress-develop/pull/6756/commits/10a638a57c91e0ed1ad3c965c44eced4acbf8b40